api/src/main/java/net/wesjd/anvilgui/AnvilGUI.java

```diff
@@ -365,7 +426,17 @@ public Builder title(String title) {
 		 */
 		public Builder item(ItemStack item) {
```

Comment:
I am not sure about this method's name, it must definitely stay because compatibility reasons, but logically there should be a counterpart to `itemRight()`.

Perhaps just make this method redirect to a `itemLeft()` and make `item()` deprecated (as a suggestion)

Reply:
Sounds good to me, which version should i specify in the docs?

Reply:
assuming latest is 1.3.0, i'll use 1.3.1